lib/std/math/big/int.zig

```diff
@@ -188,8 +197,8 @@ pub const Mutable = struct {
         if (self.limbs.ptr != other.limbs.ptr) {
             @memcpy(self.limbs[0..other.limbs.len], other.limbs[0..other.limbs.len]);
         }
-        self.positive = other.positive;
-        self.len = other.limbs.len;
+        self.positive = other.positive or other.eqlZero();
+        self.normalize(other.limbs.len);
```

Comment:
This avoid iterating over the extra zeroes twice.
```suggestion
        self.normalize(other.limbs.len);
        self.positive = other.positive or self.eqlZero();
```

Reply:
Ah, in the `eqlZero`. Good idea!